a5/core/a5_math.py

```diff
@@ -7,7 +7,7 @@
 import math
 import numpy as np
 from typing import cast
-from .types import (
+from .a5_types import (
```

Comment:
OK, fair point about the filenames not being great. I've thought about it and think this renaming is better:

- `coordinate_systems.py` (current `types.py`)
- `coordinate_transforms.py` (current `math.py`)

as the imports will read better:

```
from a5.core.coordinate_systems import Polar, Face, Cartesian
```